libcxx/include/__numeric/midpoint.h

```diff
@@ -51,19 +51,12 @@ _LIBCPP_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK
     return __a + __half_diff;
 }
 
-
 template <class _TPtr>
```

Comment:
I would suggest renaming this template parameter to `_Tp` since now this is just a normal type, not a pointer anymore.